### ISSUE TITLE
[enhancement] Icon path for Bubbles

### ISSUE THREAD

**Issue by snte** [cy33hc/vita-launcher#33]
Dear cy33hc,

first, thank you so much for the outstanding work with Vita Launcher. This really gives a second life for the Vita as a homebrew handheld, and user experience :)

One thing i would like to propose is if it would be possible to enhance the display of icons with game->type == TYPE_BUBBLE (especially PS1 Eboots), because these icon0.png are very low resolution. Maybe it would be possible to add an option to PS1, PSP categories for defining an icon_path for cover images, like for other categories (NES, SNES, GB, ...), so users can opt to use custom thumbnails also for bubbles.

Thanks in advance for your consideration!
Best regards

**Comment by singmadan** [comment]
for PS1/PSP icons, you can replace icon0.png manually in the SMLAExxxx folder (ux0:data/SMLA00001/data).

**Comment by snte** [comment]
Thanks, this is true for games in PSPEMU folder. But for PS1, PSP (and Vita) bubbles, the low-res pic0.png/icon0.png from ur0:appmeta are used. It would be nice to optionally be able to set an icon_path with custom images in config.ini, like for the other rom categories.

Here is the section in game.cpp#L846, near void LoadGameImage(Game *game), that sets the png for TYPE_BUBBLE:

```
        if (game->type == TYPE_BUBBLE && strcmp(game->category, game_categories[PS_MOBILE_GAMES].category) == 0)
        {
            sprintf(icon_path, "ur0:appmeta/%s/pic0.png", game->id);
        }
        else if (game->type == TYPE_BUBBLE)
        {
            if (strncmp(game->id, "NPXS", 4)==0)
            {
                sprintf(icon_path, "vs0:app/%s/sce_sys/icon0.png", game->id);
            }
            else
            {
                sprintf(icon_path, "ur0:appmeta/%s/icon0.png", game->id);
            }
        }
```

**Comment by cy33hc** [comment]
@snte @singmadan This has being implemented in release 3.7. Read the release notes on how to name the icon files.

https://github.com/cy33hc/vita-launcher/releases/tag/latest
